openapi.yaml

```diff
@@ -308,6 +308,8 @@ definitions:
   SearchIndex:
     type: object
     properties:
+      email:
+        type: string
```

Comment:
I think you could add `format: email` on the line below this and it will be validated by the go-openapi runtime.